app/cafe.py

```diff
@@ -0,0 +1,24 @@
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError
+
+        if not visitor["wearing_a_mask"]:
```

Comment:
```suggestion
        if not visitor.get("wearing_a_mask"):
```
wearing_a_mask may not exist into visitor